src/pages/dashboard/index.tsx

```diff
@@ -37,7 +37,7 @@ const Home: NextPage = () => {
           direction={{ base: 'column', lg: 'row' }}
         >
           <Stack pt="5" order={{ base: 1, lg: 0 }}>
-            <Heading fontSize={{ md: 'xl', lg: '2xl' }}>
+            <Heading fontSize={{ base: 'xl', md: 'xl', lg: '2xl' }}>
```

Comment:
since they both have the same value.
```suggestion
            <Heading fontSize={{ base: 'xl', lg: '2xl' }}>
```